src/login/logind-dbus.c

```diff
@@ -2102,14 +2178,18 @@ static void reset_scheduled_shutdown(Manager *m) {
         m->nologin_timeout_source = sd_event_source_unref(m->nologin_timeout_source);
 
         m->scheduled_shutdown_type = NULL;
+        m->scheduled_shutdown_timeout = USEC_INFINITY;
+        m->scheduled_shutdown_uid = UID_INVALID;
+        freep(&m->scheduled_shutdown_tty);
+        freep(&m->wall_message);
```

Comment:
nope. please do note use freep like this. it's used for _cleanup_ only. Use mfree() insted:

```c
m->scheduled_shutdown_tty = mfree(m->scheduled_shutdown_tty);
```

Reply:
#22672